app/src/main/java/com/github/braillesystems/learnbraille/ui/views/BrailleDotsView.kt

```diff
@@ -19,7 +19,7 @@ import org.koin.core.KoinComponent
 import org.koin.core.inject
 import timber.log.Timber
 
-class BrailleDotView : CheckBox {
+class BrailleDotView : androidx.appcompat.widget.AppCompatCheckBox {

```

Comment:
maybe this change causes error:

![photo_2020-09-25_22-05-31](https://user-images.githubusercontent.com/25281147/94307156-e880bc80-ff7c-11ea-9792-92f1b35674cf.jpg)

Reply:
This patch fixes issue:
```
@SuppressLint("AppCompatCustomView") // Causes BrailleDotView misplacement
class BrailleDotView : CheckBox {
```

Reply:
Of course, it will be better to migrate to app compat version, but it could be done later. So issue about it should be open.

Reply:
I've fixed (applied a patch ⬆️ )

and added a point in the "Development" list in issue https://github.com/braille-systems/learn-braille/issues/257 (this doesn't mean I do not want it to be done, we may later create an issue for this)